src/test/java/com/ss/ita/google/ChangeLinksColorTest.java

```diff
@@ -1,28 +1,27 @@
 package com.ss.ita.google;
 
-import com.ss.ita.google.ui.pages.HomePage;
-import com.ss.ita.google.ui.pages.SearchResultPage;
+import com.ss.ita.google.ui.pages.*;
 import com.ss.ita.google.ui.pages.runnerAndProperties.TestRunner;
 import org.testng.annotations.Test;
 
+import static com.ss.ita.google.ui.pages.util.WebElementUtil.*;
 import static org.testng.Assert.*;
 
 public class ChangeLinksColorTest extends TestRunner {
 
     @Test
     public void verifyChangingLinksColor() {
+        int numberOfSearchedResultLink = 0; //first link to get text and color.
+
         SearchResultPage searchResultPage = new HomePage()
                 .doSearch("Funny kitten picture");
         String firstLinkText = searchResultPage
-                .getLinkText(0);
-        String defaultColor = searchResultPage
-                .getLinkColor(0);
-        searchResultPage.changeFirstLinkColor("lightpink");
-        String changedColor = searchResultPage
-                .getLinkColor(0);
+                .getSearchedResultLinkText(numberOfSearchedResultLink);
+        String defaultColor = getColor(searchResultPage.getSearchedResultLink(numberOfSearchedResultLink));
+        setColorViaJs(searchResultPage.getSearchedResultLink(numberOfSearchedResultLink), "lightpink");
+        String changedColor = getColor(searchResultPage.getSearchedResultLink(numberOfSearchedResultLink));
```

Comment:
searchResultPage.getSearchedResultLink(numberOfSearchedResultLink)) - this is duplicated 3 times

Reply:
you said that if we use a method here we must to create object 3 times and its bad and do it in test. Or I don't undestand something?
![image](https://user-images.githubusercontent.com/90624987/140025884-555c11db-34b6-4aaf-9274-4c73f4defe38.png)